examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -116,16 +144,21 @@ const ExternalDataView: React.FC<IExternalDataViewProps> = (props: IExternalData
 };
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface ISyncStatusViewProps { }
+interface ISyncStatusViewProps {
+    fluidSync: boolean;
+    unsynchronizedChangesCount: number;
+    handleCountUpdate: (count: number) => void;
+}
 
 // TODO: Implement the statuses below
 const SyncStatusView: React.FC<ISyncStatusViewProps> = (props: ISyncStatusViewProps) => {
+    useEffect(() => { }, [props.unsynchronizedChangesCount, props.fluidSync]);
     return (
         <div>
             <h3>Sync status</h3>
             <div style={{ margin: "10px 0" }}>
-                Fluid has [no] unsync'd changes (not implemented)<br />
-                External data source has [no] unsync'd changes (not implemented)<br />
+                Fluid has {props.fluidSync ? "some" : "no"} unsync'd changes.<br />
```

Comment:
would be careful here with javascript falsey value evaluation. E.g. 0, '0', undefined, null 'false' will all be evaluated as the equivalent of boolean false.

Reply:
The value itself should be a boolean that gets toggled when the fluid Container detects valueChanged events and then toggled back when those changes are saved.